packages/sdks/src/blocks/columns/columns.lite.tsx

```diff
@@ -41,6 +41,19 @@ export default function Columns(props: ColumnProps) {
     get stackAt() {
       return props.stackColumnsAt || 'tablet';
     },
+    get blocksStyle() {
+      return useTarget({
+        solid: {
+          'flex-grow': '1',
+        },
```

Comment:
@samijaber I've made this as a state variable, but it looks like it should be handled in mitosis.

if style prop is provided as a state variable or object, we can wrap it up inside `convertCamelToKebab(styles)`

Reply:
Yeah, could be a good mitosis improvement to have ootb.

To avoid unnecessary state, could we inline this `useTarget` in `styleProp={useTarget(/**/)}`?